### PR TITLE
Effector UX improvements

### PR DESCRIPTION
**Pull request by algairim**
Signed-off-by: Mykola Mandra <mykola.mandra@cloudsoft.io>

### CODE REVIEW

File: ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html [hunk 1]
```diff
@@ -59,8 +59,37 @@ <h3>Parameters</h3>
...
+                        tooltip-trigger="{'copied': 'mouseleave'}" tooltip-placement="left"
+                        tooltip-append-to-body="true">
+                    <i class="fa fa-clipboard"></i>
+                </button>
+            </span>
+            <pre>{{brCmd()}}</pre>
+        </div>
+    </section>
+    <section ng-if="vm.cli">
+        <h3>REST</h3>
```
Comment (jcabrerizo) [inline ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html:78]: Think `API` could it more clear for non-dev Brooklyn operators

**Review by jcabrerizo — APPROVED** [review]
Looks great @algairim , I added a minor comment, fell free to ignore it and merge. Thanks

File: ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html (hunk 1 above)
Comment (algairim) [reply ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html:78]: I'll keep `REST`, as it was agreed earlier.

Comment (algairim) [reply ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html]: Changing it to API as there are 2 votes for it.

**Pull request merged**